rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf

```diff
@@ -133,7 +132,7 @@ SecRule FILES|FILES_NAMES "!@rx (?i)^(?:&(?:(?:[acegiln-or-suz]acut|[aeiou]grav|
 # is NOT all digits, then it will match.
 #
 # -=[ References ]=-
-# https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.13
+# https://www.rfc-editor.org/rfc/rfc9110.html
```

Comment:
Same here. More specific URL pointing to the section?

Reply:
Fixed in [c725699](https://github.com/coreruleset/coreruleset/pull/3232/commits/c725699d2bb4c7f4e7d968a9eab8e4099d304add)